Google.PowerShell/Common/GcloudWrapper.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Google.PowerShell.Common
+{
+    /// <summary>
+    /// This class wraps the gcloud command and offers up some of its services.
```

Comment:
"wraps the gcloud command" isn't especially specific. How about:

"This class shell executes "gcloud {command} --format=json", to allow delegation to Cloud SDK implementation. Used for things like credential management."